htdocs/compta/resultat/clientfourn.php

```diff
@@ -654,7 +654,7 @@
 	if (! empty($date_start) && ! empty($date_end))
 		$sql.= " AND $column >= '".$db->idate($date_start)."' AND $column <= '".$db->idate($date_end)."'";
 
-		$sql.= " GROUP BY p.rowid, p.ref, u.firstname, u.lastname, dm";
+		$sql.= " GROUP BY u.rowid";
```

Comment:
A lot of field are missing in the group by. This will make error with postgresql. We should find same fields than into select except sum()